src/libostree/ostree-bootloader-zipl.c

```diff
@@ -80,6 +83,203 @@ _ostree_bootloader_zipl_write_config (OstreeBootloader  *bootloader,
   return TRUE;
 }
 
+
+
+static gboolean
+_ostree_secure_execution_get_keys (GArray **keys, GError **error)
+{
+  const char *keyspath = "/etc";
+  g_autoptr(GFile) path          = g_file_new_for_path (keyspath);
+  g_autoptr(GFileEnumerator) etc = g_file_enumerate_children (path, G_FILE_ATTRIBUTE_STANDARD_NAME,
+                                                              G_FILE_QUERY_INFO_NONE, NULL, error);
+  if (*error)
+    {
+      sd_journal_print(LOG_ERR, "s390x SE: looking for SE keys: %s", (*error)->message);
+      return FALSE;
+    }
+  *keys = g_array_new (FALSE, FALSE, sizeof(GString*));
+  while (1) {
+      g_autoptr(GFileInfo) nfo = g_file_enumerator_next_file (etc, NULL, NULL);
+      if (!nfo)
+        break;
+      const char *name = g_file_info_get_name (nfo);
+      if (g_str_has_prefix(name, SECURE_EXECUTION_KEY_PREFIX)) {
+          GString *fullname = g_string_new ("");
+          g_string_printf (fullname, "%s/%s", keyspath, name);
+          g_array_append_val (*keys, fullname);
+          sd_journal_print(LOG_INFO, "s390x SE: found key: %s", fullname->str);
```

Comment:
In general we've been conservative with logging since ostree is a library.  Fine to keep for now, though I'd also say it's probably simpler to log all the keys in the caller instead.